openpype/hosts/blender/scripts/build_workfile.py

```diff
@@ -874,6 +875,104 @@ def build_lipsync(project_name: str, shot_name: str):
             )
 
 
+def build_fabrication(project_name: str, asset_name: str):
+    """Build fabrication workfile.
+
+    Args:
+        project_name (str): The current project name from OpenPype Session.
+        asset_name (str): The current asset name from OpenPype Session.
+    """
+
+    # Get subsets
+    asset_doc = get_asset_by_name(
+        project_name, "LightSetupBank", fields=["_id"]
+    )
+    subsets = get_subsets(
+        project_name,
+        asset_ids={asset_doc["_id"]},
+        fields=["_id", "name", "data"],
+    )
+
+    representations = []
+    world_name = ""
+    # Match and download and load subset
+    for subset in subsets:
+        raw_asset_name = asset_name.split("_")[0].lower()
+        raw_subset_name = (
+            subset["name"].strip("lighting").rsplit("_", 1)[0].lower()
+        )
+        if raw_asset_name == raw_subset_name:
+            representation = download_subset(
+                project_name,
+                "LightSetupBank",
+                subset["name"],
+            )
+            if "world" in subset["name"].lower():
```

Comment:
I don't understand how this can work, no any `LightSetupBank` subset is called with "world" in its name.

Reply:
The variable `world_name` is set in this condition and i use `world_name` to apply it, and strangely it worked so i don't know why but it seems a world is imported in the subsets